pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -155,9 +168,37 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		return nil, kapierror.NewInternalError(err)
 	}
 
+	// if there's no rolebinding in the project, go ahead and return
+	if lastRoleBinding == nil {
```

Comment:
> nit: check for non-nil

This direction is the "immediate return".  The other direction is "do a bunch more work.